### PR TITLE
Add Acknowledgement for RICG work

### PR DESCRIPTION
**Pull request by travisleithead**
Address issue #286

### CODE REVIEW

File: sections/acknowledgements.include [hunk 1]
```diff
@@ -9,6 +9,14 @@
   and the other editors of the WHATWG: Anne van Kesteren, Domenic Denicola, Philip Jägenstedt,
   Simon Pieters.
 
+  Thanks to the participants of the 
+  <a href="http://www.w3.org/community/respimg/participants">Responsive Images Community Group</a> 
+  and the WHATWG for helping to develop the <{picture}> element, the <{img/srcset}> attribute, 
+  and the <{source/sizes}> attribute. Special thanks to Bruce Lawson for originally suggesting ,
```
Comment (chaals) [inline sections/acknowledgements.include]: s/ ,/,/ at the end of the line

**Comment by travisleithead** [comment]
Good catch. ❤️ peer review.

File: sections/acknowledgements.include [hunk 2]
```diff
@@ -9,6 +9,14 @@
   and the other editors of the WHATWG: Anne van Kesteren, Domenic Denicola, Philip Jägenstedt,
   Simon Pieters.
 
+  Thanks to the participants of the 
+  <a href="http://www.w3.org/community/respimg/participants">Responsive Images Community Group</a> 
+  and the WHATWG for helping to develop the <{picture}> element, the <{img/srcset}> attribute, 
+  and the <{source/sizes}> attribute. Special thanks to Bruce Lawson for originally suggesting, 
+  Edward O'Connor and Ian Hickson for writing the original srcset spec, and Adrian Bateman for
```
Comment (arronei) [inline sections/acknowledgements.include]: Can we change spec to specification?

Do we also want to link to that document using [[SRCSET]]

Comment (travisleithead) [inline sections/acknowledgements.include]: Done.

**Pull request merged**